migrations/1657807941757-market-currency.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('lodash/fp')
+const { loadLatest, saveAccounts } = require('../lib/new-settings-loader')
+const { ACCOUNT_LIST } = require('../lib/new-admin/config/accounts')
+const { ALL } = require('../lib/plugins/common/ccxt')
+
+exports.up = function (next) {
+  return loadLatest()
+    .then(({ config, accounts }) => {
+      const allExchanges = _.map(it => it.code)(_.filter(it => it.class === 'exchange', ACCOUNT_LIST))
+      const configuredExchanges = _.intersection(allExchanges, _.keys(accounts))
+      const localeCurrency = config.locale_fiatCurrency
+
+      const newAccounts = _.reduce(
+        (acc, value) => {
+          if (!_.isNil(accounts[value].currencyMarket)) return acc
+          if (_.includes(localeCurrency, ALL[value].FIAT)) return { ...acc, [value]: { currencyMarket: localeCurrency } }
+          return { ...acc, [value]: { currencyMarket: _.head(ALL[value].FIAT) } }
+        },
+        {},
+        configuredExchanges
+      )
```

Comment:
Isn't this changing the default behaviour? If we used to default to EUR, why are we not defaulting to EUR here? 

Relying on the FIAT order for default fiat currency might be problematic as well. Maybe create a new variable like DEFAULT_FIAT_MARKET.